src/psyclone/domain/gocean/transformations/gocean_loop_fuse_trans.py

```diff
@@ -40,7 +40,6 @@
 
 from psyclone.psyir.transformations import LoopFuseTrans, TransformationError
 import psyclone.gocean1p0
-import psyclone.gocean0p1
 
```

Comment:
Copyright date please.

Reply:
Done, and converted to fstring since there was just 2 to make this file pylint clean